typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -292,112 +293,122 @@ const existingProxyAdmins: ChainMap<{ address: string; owner: string }> = {
   },
 };
 
-export const getRenzoEZETHWarpConfig = async (): Promise<
-  ChainMap<HypTokenRouterConfig>
-> => {
-  const config = getEnvironmentConfig('mainnet3');
-  const multiProvider = await config.getMultiProvider();
-  const registry = await getMainnet3Registry();
+export function getRenzoEZETHWarpConfigGenerator(
+  ezEthSafes: Record<string, string>,
```

Comment:
Shouldn't this be the same shape as xERC20?

Reply:
yis. fixed